packetery/src/Packetery/Plugin.php

```diff
@@ -279,13 +278,30 @@ public function add_menu_pages(): void {
 	}
 
 	/**
-	 * Loads plugin translations files.
+	 * Gets current locale.
 	 */
-	public function load_textdomains(): void {
-		$mo_files = Finder::findFiles( '*.mo' )->from( PACKETERY_PLUGIN_DIR . '/languages' );
-		foreach ( $mo_files as $mo_file ) {
-			load_textdomain( 'packetery', $mo_file );
+	private function getLocale(): string {
+		return apply_filters(
+			'plugin_locale',
+			( ( is_admin() && function_exists( 'get_user_locale' ) ) ? get_user_locale() : get_locale() ),
```

Comment:
Není toto zbytečné ? function_exists( 'get_user_locale' ) ) ? get_user_locale() : get_locale()
Proč se testuje zda funkce get_user_locale zda funkce existuje ?

Reply:
Souvisí to s verzí WC / WP.

Reply:
To tvrdí dokumentace. Já dokumentacím moc nevěřím.